init.ps1

```diff
@@ -13,7 +13,11 @@ param(
     [boolean]$addOctopack = $true,
     [boolean]$addUnitTests = $true,
     [boolean]$addRebuildDatabase = $true,
-    [string]$branch = "master"
+    [string]$branch = "master",
+	[boolean]$importTemplateRepo = $true,
+	[string]$repoName = "ClearMeasureBootcamp",

```

Comment:
I had originally had my repo as the default. https://github.com/JustinMason/BaselineSolution
I put the boot camp one in there to help Jeffrey see how this project could handle his desires as well as the future templates that could be created.